src/webchat-ui/components/presentational/previous-conversations/PreviousConversationsOptions.tsx

```diff
@@ -0,0 +1,154 @@
+import styled from "@emotion/styled";
+import { IWebchatConfig } from "../../../../common/interfaces/webchat-config";
+import React from "react";
+import { Typography } from "@cognigy/chat-components";
+import IconButton from "../IconButton";
+import Modal from "../../Modal/Modal";
+import Button from "../Button";
+import { useSelector, useDispatch } from "react-redux";
+import { setConversations } from "../../../../webchat/store/previous-conversations/previous-conversations-reducer";
+import { getStorage } from "../../../../webchat/helper/storage";
+import { setShowPreviousConversationsOptionsScreen } from "../../../../webchat/store/ui/ui-reducer";
+import { StoreState } from "../../../../webchat/store/store";
+
+const Container = styled.div`
+	display: flex;
+	flex-direction: column;
+`;
+
+const OptionsContainer = styled.div`
+	display: flex;
+	padding: 10px 20px;
+	flex-direction: column;
+`;
+
+const DeleteButtonContainer = styled.div(() => ({
+	display: "flex",
+	alignItems: "flex-start",
+	gap: 8,
+}));
+
+const DeleteButton = styled(IconButton)(({ theme }) => ({
+	background: theme.red,
+	display: "flex",
+	padding: "12px 16px",
+	justifyContent: "center",
+	alignItems: "center",
+	gap: 16,
+	borderRadius: 15,
+	color: theme.white,
+	"&: focus-visible": {
+		outline: `2px solid ${theme.primaryColor}`,
+		outlineOffset: 2,
+	},
+}));
+
+const CancelButton = styled(Button)(({ theme }) => ({
+	color: theme.black10,
+	width: "auto",
+	display: "flex",
+	padding: "12px",
+	borderRadius: 12,
+	border: `1px solid ${theme.greyColor}`,
+	"&: focus-visible": {
+		outline: `2px solid ${theme.primaryColor}`,
+		outlineOffset: 2,
+	},
+}));
+
+const DeleteAnywaysButton = styled(Button)(({ theme }) => ({
+	color: theme.white,
+	background: theme.red,
+	width: "auto",
+	padding: "12px",
+	borderRadius: 12,
+	"&: focus-visible": {
+		outline: `2px solid ${theme.primaryColor}`,
+		outlineOffset: 2,
+	},
+	"&:hover:not(:disabled)": {
+		backgroundColor: theme.red10,
+		color: theme.black10,
+	},
+}));
```

Comment:
Is it possible to reuse the primary/secondary buttons that we already use in other places? This way we can easily maintain all these buttons and we do not have to style them individually.